src/pages/documentation/adobe-audience-manager/api-reference.md

```diff
@@ -8,17 +8,9 @@ The `extensionVersion()` API returns the version of the Audience extension that
 
 To get the version of the Audience extension, use the following code sample:
 
-<TabsBlock orientation="horizontal" slots="heading, content" repeat="8"/>
```

Comment:
This info is incorrect, this SDK is only available in Android, iOS and React Native, the other tabs should be removed